curr_scraper/apps/api_caller/models.py

```diff
@@ -11,9 +11,10 @@ class Meta:
 
 class CurrencyRateRecord(models.Model):
     timestamp = models.TimeField()
-    curr = models.ForeignKey(Currency, on_delete=models.PROTECT, related_name='%(class)s_currency')
-    base_curr = models.ForeignKey(Currency, on_delete=models.PROTECT, related_name='%(class)s_base_currency')
-    rate = models.FloatField()
+    curr = models.ForeignKey(Currency, on_delete=models.PROTECT, related_name='currency_name_records_currency')
+    base_curr = models.ForeignKey(Currency, on_delete=models.PROTECT,
```

Comment:
Also, please add verbose name for each field with translation support.
Please check this StackOverflow question: https://stackoverflow.com/questions/35594839/django-verbose-name-translations